pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -14,19 +14,21 @@ def setUp(self):
     def _expected_command(self, expected_text_append, expected_default_store=None):
         if expected_text_append:
             expected_text_append = "/" + expected_text_append
+        shard = os.environ.get('SHARD')
         expected_statement = (
             "DEFAULT_STORE={default_store} "
-            "SCREENSHOT_DIR='{repo_dir}/test_root/log' "
-            "BOK_CHOY_HAR_DIR='{repo_dir}/test_root/log/hars' "
-            "SELENIUM_DRIVER_LOG_DIR='{repo_dir}/test_root/log' "
+            "SCREENSHOT_DIR='{repo_dir}/test_root/log{shard}' "
```

Comment:
Looks like this will appear as `log3` for example? Should we have an underbar or other space-filler?